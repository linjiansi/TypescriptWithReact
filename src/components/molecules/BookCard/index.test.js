import React from 'react';
import renderer from 'react-test-renderer';
import { BookCardComponent } from './index';

it('renders correctly at BookCardComponent', () => {
  const tree = renderer
    .create(<BookCardComponent bookName='test'
                               bookPrice={1000}
                               purchaseDate='2021/05/25'></BookCardComponent>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
