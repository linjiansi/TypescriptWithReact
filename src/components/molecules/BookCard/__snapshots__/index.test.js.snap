// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly at BookCardComponent 1`] = `
<a
  href="/edit"
  onClick={[Function]}
>
  <div
    className="MuiPaper-root MuiCard-root makeStyles-root-1 MuiPaper-elevation1 MuiPaper-rounded"
  >
    <div
      className="MuiCardMedia-root makeStyles-media-2"
      style={
        Object {
          "backgroundImage": "url(\\"https://source.unsplash.com/random\\")",
        }
      }
      title="book"
    />
    <div
      className="MuiCardContent-root"
    >
      <p
        className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextPrimary"
      >
        書籍名: 
        test
      </p>
      <div
        className="sc-bdnxRM ctBBVd"
      >
        <p
          className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
        >
          金額: 
          1000
        </p>
        <p
          className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
        >
          購入日: 
          2021/05/25
        </p>
      </div>
    </div>
  </div>
</a>
`;
